apps/angular/src/app/features/anime/manage-anime/edit-anime-page/edit-anime-page.component.ts

```diff
@@ -0,0 +1,45 @@
+import { ChangeDetectionStrategy, Component, inject } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { AnimeService } from '@js-camp/angular/core/services/anime.service';
+import { startLoadingStatus } from '@js-camp/angular/core/utils/loader-starter';
+import { stopLoadingStatus } from '@js-camp/angular/core/utils/loader-stopper';
+import { AnimeDetail } from '@js-camp/core/models/anime/anime-detail';
+import { BehaviorSubject, Observable, map, switchMap } from 'rxjs';
+
+/** Edit anime page. */
+@Component({
+	selector: 'camp-edit-anime-page',
+	templateUrl: './edit-anime-page.component.html',
+	styleUrls: ['./edit-anime-page.component.css'],
+	changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class EditAnimePageComponent {
```

Comment:
```suggestion
export class AnimeEditComponent {
```
Class names should be noun, not verb